docs/metrics/getting-started/README.md

```diff
@@ -87,9 +87,11 @@ the configuration for metrics like `Meter` names, readers, etc. and is highly
 ## OpenTelemetry .NET special note
 
 Metrics in OpenTelemetry .NET is a somewhat unique implementation of the
-OpenTelemetry project, as most of the Metrics API are incorporated directly
```

Comment:
Change ".NET Core SDK" to ".NET SDK" in line 3?

Reply:
Thanks. a good change for a next PR.